imap_processing/mag/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = "01"
```

Comment:
I don't know if you use this version for filename. If you do, this is not in the format it expects and will throw error.

Reply:
No, this is the version for the instrument code, which I believe only gets written into the CDF files. The CLI gets a version passed in, and does not generate versions at all.